meta/pattern-template.md

```diff
@@ -1,32 +1,38 @@
 ## Title  
 
 ## Problem
-What is the problem - crisp definition of the problem
+What is the problem - crisp definition of the problem. 1-2 sentances. Anymore should go into other sections.
 
 ## Context
-Where does the problem exist? what are the pre-conditions?
+Where does the problem exist? what are the pre-conditions? **Unchangable** before the solution goes into place. The content here is often tied to applicability of other readers: "Do I have this same particular situation?"
 
 ## Forces
-What makes the problem difficult? What are the trade-offs? These are constraints that can be changed at a cost.
+What makes the problem difficult? What are the trade-offs? These are constraints that **can be changed** at a cost. The solution might change one or more of these forces in order to solve the problem, while also in-turn changing the context.
 
 ## Sketch (optional)
 visual illustration
 
 ## Solutions
-Verified resolutions and possible resolutions to the problem
+Verified resolutions and possible resolutions to the problem.
 
 ## Desired Resulting Context
```

Comment:
Should this just be
## Resulting Context

It's only Desired Resulting Context in the case of a donut, no?